src/sendmail/mail_sender.py

```diff
@@ -29,6 +37,7 @@ def send_email(records: list[BuildingServicesSearchResult]):
     mail = mt.Mail(
         sender=mt.Address(email=MAILTRAP_SENDER_ADDRESS, name="Code For Dayton"),
         to=[mt.Address(email=MAILTRAP_TO_ADDRESS)],
+        cc=[mt.Address(email=MAILTRAP_CC_ADDRESS)],
```

Comment:
This causes an error when a cc address is not configured because it is optional

Reply:
does this do the same for BCC as well?

Reply:
Yeah, it doesn't like being passed the Address object without an actual address set